Gaijinizer patch/Map125.txt

```diff
@@ -170,8 +177,8 @@ Hmm? What kind of anta are you?#MTLed
  普段は行商人やってる、しがないオニさ。
 > CONTEXT: Map125/events/9/pages/0/88/Dialogue/hyakki
 > CONTEXT: Map125/events/9/pages/1/88/Dialogue/hyakki
-'I' is Hyakki. I am usually a hawker, there is no
-onion.#MTLed
+The name's Hyakki. I usually work as a peddler,
+just an insignificant oni.#Ron
 > END STRING
```

Comment:
I feel like rewording this to be less stiff:
>_**The name's Hyakki. I'm nothing more
then an unimportant oni peddler.**_